blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeImpl.java

```diff
@@ -67,6 +69,18 @@ public PipelineNodeImpl(FlowNodeWrapper node, Reachable parent, WorkflowRun run)
         this.durationInMillis = node.getTiming().getTotalDurationMillis();
         this.self = parent.getLink().rel(node.getId());
         this.parent = parent;
+        // TODO move to use of https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/596
```

Comment:
https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/596

There is an incremental deployment you could pick up (if this were switched to draft).

Reply:
can't really as target branch is `1.25.5.x`